src/Core/RenderAPI/EspPipelineBuilder.cc

```diff
@@ -0,0 +1,30 @@
+#include "EspPipelineBuilder.hh"
+
+#include "Platform/Vulkan/VulkanPipelineBuilder.hh"
+
+namespace esp
+{
+  EspVertexAttribute::EspVertexAttribute(uint32_t location, EspAttrFormat format, uint32_t offset) :
+      m_location(location), m_format(format), m_offset(offset)
```

Comment:
I'd appreciate initialising field with curly brackets (I mean "{" and "}") for consistency